src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -8,6 +8,7 @@ Release History
 * vm: expose 'az vm list-skus' command
 * vm/vmss: support to assign identity w/o creating role assignments
 * vm: apply storage sku on attaching data disks
+* vm: remove default os-disk name and storage SKU when using managed disks.
```

Comment:
You might need another entry for VMSS which is also impacted by the storage sku change